nodes/Assign.py

```diff
@@ -1 +1,13 @@
+"""
+Assign astroid node
+
+Attributes:
+    - targets  (list)  a list of nodes
+    - value    (type)  a single node
```

Comment:
This should probably be an arbitrary `Node` or at least `Expr` class, and definitely not "type" (since you're assigning a value to a name).